lib/main.dart

```diff
@@ -1,125 +1,145 @@
-    );
+        theme: ThemeData(
+          colorScheme: ColorScheme.fromSwatch().copyWith(
+            primary: Colors.black12,
+          ),
+        ),
+        home: Scaffold(
+            backgroundColor: Colors.cyan,
+            appBar: AppBar(
+              title: Text(
+                m.title,
+              ),
+              centerTitle: true,
+            ),
+            body: SingleChildScrollView(
+              child: SafeArea(
+                  top: true,
+                  bottom: true,
+                  left: true,
+                  right: true,
+                  child: Center(
+                    child: Column(
+                      children: [
+                        Stack(
+                          children: [
+                            Image.network(m.backdrop),
+                            Positioned(
+                              bottom: 0,
+                              right: 20,
+                              child: Text(
+                                m.originalTitle,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 25,
+                                  color: Colors.white,
+                                ),
+                                textAlign: TextAlign.center,
+                              ),
+                            ),
+                          ],
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                            children: [
+                              MiddleText(s: 'Available on ${m.releaseDate}'),
+                              MiddleText(s: m.voteAverage),
+                            ],
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: m.getGenreElements(),
+                        ),
+                        const SizedBox(
+                          height: 20,
+                        ),
+                        Stack(
+                          children: [
+                            Image.network(m.poster),
+                            const Positioned(
+                              top: 15,
+                              right: 15,
+                              child: Column(
+                                children: [
+                                  Icon(
+                                    Icons.favorite_outlined,
+                                    size: 55,
+                                    color: Colors.red,
+                                  ),
+                                  FloatingActionButton(
+                                    onPressed: null,
```

Comment:
you can set a void function to have the button active (){}, instead of null

Reply:
done